bin/apib2swagger.js

```diff
@@ -123,28 +143,34 @@ function processBlueprint(blueprint, opts) {
     });
 }
 
+function serveFile(response, path){
+    response.statusCode = 200;
+    response.write(fs.readFileSync(path));
+    response.end();
+    return;
+}
+
 function runServer(swagger) {
     var server = http.createServer(function(request, response) {
-        console.log(request.url);
-        var path = request.url.split('?')[0];
-        if (path === '/swagger.json') {
+        var filePath = request.url.split('?')[0];
+        if (filePath === '/swagger.json') {
             response.statusCode = 200;
             response.write(JSON.stringify(swagger));
             response.end();
-        } else if (path === '/') {
+        } else if (filePath === '/') {
             response.statusCode = 302;
             response.setHeader('Location', '/index.html?url=/swagger.json');
             response.end();
         } else {
-            var file = 'swagger-ui-master/dist' + path;
-            if (!fs.existsSync(file)) {
-                response.statusCode = 404;
-                response.end();
-                return;
-            }
-            response.statusCode = 200;
-            response.write(fs.readFileSync(file));
+            const swaggerFile = 'swagger-ui-master/dist' + filePath;
+            if (fs.existsSync(swaggerFile)) return serveFile(response, swaggerFile)
+
+            const file = util.getAbsolutePath(options, filePath)
+            if (file && fs.existsSync(file)) return serveFile(response, file)
```

Comment:
Because `util.getAbsolutePath` always returns existing files, this `fs.existsSync` is unnecessary.

Reply:
Yes, I did consider that but I will go ahead and remove the extra check.

Reply:
Done